docs/minishift-installation/master.adoc

```diff
@@ -9,29 +9,30 @@ include::topics/templates/document-attributes.adoc[]
 :context: minishift
 
 [.lead]
-This guide walks you through the process of installing the {name-launcher} tool to run on a local cloud as provisioned by a {OpenShiftLocal}.
-This includes link:https://www.openshift.org/minishift/[{Minishift}], an all-in-one VM with link:https://www.okd.io[OKD^], or link:https://developers.redhat.com/products/cdk/overview/[{CDK}], a VM that includes OpenShift Container Platform.
+This guide walks you through the process of installing the {name-launcher} tool on an {OpenShiftCluster}.
+To test the launcher tool locally on your machine
```

Comment:
The link is commented out, and so is not visible in the rendered version. Please check.

Reply:
Thank you for catching this. I can fix this, but I need to re-create the link, because I changed the original chapter:)